src/frontend.cs

```diff
@@ -1137,7 +1137,7 @@ public override object VisitVarPostOpAssign(bhlParser.VarPostOpAssignContext ctx
       FireError(Location(ctx.NAME()) + " : symbol not resolved");
 
     if(SymbolTable.IsRelopCompatibleType(vlhs.type.type) == false)
```

Comment:
Why not:

if(!SymbolTable.IsRelopCompatibleType(vlhs.type.type))

?

Reply:
symbol "!" difficult to notice among the rest,
I considered an explicit comparison cleaner

Reply:
Well, this style is not in coding conventions for this project, sorry. I'd like to keep the codebase consistent.

Reply:
ok